src/main/java/com/ibm/stocator/fs/swift/SwiftAPIDirect.java

```diff
@@ -101,13 +101,12 @@ public static SwiftInputStreamWrapper getObject(Path path, JossAccount account,
       httpGet.addHeader(Constants.RANGES_HTTP_HEADER, rangeValue);
     }
     httpGet.addHeader(Constants.USER_AGENT_HTTP_HEADER, Constants.STOCATOR_USER_AGENT);
-    CloseableHttpClient httpclient = HttpClients.createDefault();
-    CloseableHttpResponse response = httpclient.execute(httpGet);
+    HttpClient httpclient = account.getHttpClient();
+    HttpResponse response = httpclient.execute(httpGet);
     int responseCode = response.getStatusLine().getStatusCode();
```

Comment:
I changed this because I created a single instance of HttpClient to share in JossAccount and needed to use that one to execute the request.